lib/src/useRedundantClick.ts

```diff
@@ -38,9 +38,19 @@ const useRedundantClick = <T extends HTMLElement = HTMLElement>() => {
     targetRef.current.dispatchEvent(newEvent);
   };
 
+  // Add to the container element
+  const handleAuxiliaryClick = (event: React.MouseEvent) => {
+    if (event.button !== 1) {
+      return;
+    }
```

Comment:
This is so that the handler specifically only handles middle mouse clicks. I believe if we removed it, it would start firing on the other mouse buttons as well (right click, browser back, browser forward), as per this list: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button#value (since the `auxclick` event fires for all non-primary mouse buttons)